irc/help.go

```diff
@@ -45,6 +45,8 @@ Ergo supports the following channel modes:
   +i  |  Invite-only mode, only invited clients can join the channel.
   +k  |  Key required when joining the channel.
   +l  |  Client join limit for the channel.
+  +f  |  Users who are unable to join this channel (due to +i or +l) are forwarded
```

Comment:
```suggestion
  +f  |  Users who are unable to join this channel (due to another mode) are forwarded
```

I think +R counts too and there may be more or could be more modes in the future so I don't know if they need to be listed individually. I think it doesn't affect to bans though so my wording may not be the best one either.